pkg/apis/platform/v1/groupversion_info.go

```diff
@@ -0,0 +1,39 @@
+/*
+Copyright 2020 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Package v1 contains API Schema definitions for the platform v1 API group
+// +kubebuilder:object:generate=true
+// +groupName=platform.k8s.flanksource.com
+package v1
+
+import (
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"sigs.k8s.io/controller-runtime/pkg/scheme"
+)
+
+var (
+	// GroupName is the group name use in this package
+	GroupName = "platform.k8s.flanksource.com"
```

Comment:
I think maybe this might be a bit too verbose, perhaps just `flanksource.com`

Reply:
What about stripping out `k8s` and leave only `platform.flanksource.com`?

Just imagine the case that we could launch another operator. In that case, Having a different group name would be beneficial.

Reply:
Renamed the group domain to `platform.flanksource.com`